src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
```

Comment:
- As required, I Kindly suggest that you import booksSlice from `src/redux/books` and also
`Imports categoriesSlice from src/redux/categories`.